modules/functions/src/metrics/index.ts

```diff
@@ -0,0 +1,18 @@
+import { MetricType } from '@conduitplatform/grpc-sdk';
+
+export default {
```

Comment:
add another metric to measure function execution times for each function. The function name or id will be defined via labels on the metric